### ISSUE TITLE
No controls when casting

### ISSUE THREAD

**Issue by anacrolix** [arranlomas/Trickl#17]
After successfully casting to the Chromecast, there are no obvious controls available anywhere. I can't see where to pause/play/etc.

**Comment by arranlomas** [comment]
Do you not see either of the following?

![chromecast_controller_expanded](https://user-images.githubusercontent.com/22332838/32519361-9bfb2878-c460-11e7-8fa3-8a620507d073.jpg)
![chromecast_controller_collapsed](https://user-images.githubusercontent.com/22332838/32519364-9f8b923e-c460-11e7-8d78-ef17fdde05c9.jpg)

**Comment by arranlomas** [comment]
@anacrolix any update on this, I can't replicate? It doesn't show the controller when you are connected but nothing is streaming. Maybe that should be the actual behaviour. This is how Netflix behaves

**Comment by anacrolix** [comment]
Let me try again soon. My internet situation is fiddly out here. Thanks for the patience.

**Comment by arranlomas** [comment]
@anacrolix Any luck? Ill close this as I can't replicate it not showing

**Comment by anacrolix** [comment]
Yeah no problem, I had another go and didn't see the problem.

**Comment by arranlomas** [comment]
still occuring
